### ISSUE TITLE
package imported without types errors when using with Go 1.18

### ISSUE THREAD

**Issue by joshmgross** [fatih/faillint#31]
When using this tool with Go 1.18, I'm seeing errors like
```
faillint: internal error: package "time" without types was imported from "github.com/github/my-project-repo/constants"
```

This might be fixed by bumping `golang.org/x/tools` to a newer version

**Comment by joshmgross** [comment]
@fatih I'm still seeing this issue with the release version https://github.com/fatih/faillint/releases/tag/v1.9.0

The only difference I can think of is the release version being built with a different version of Go:
```bash
❯ /usr/local/bin/faillint -V 
faillint version 1.9.0 (2022-05-10T06:59:18Z)
❯ go version /usr/local/bin/faillint 
/usr/local/bin/faillint: go1.17.9
```

Meanwhile the version I built from my changes in #32 is using go1.18.1:
```bash
❯ go version /usr/local/bin/updated-faillint 
/usr/local/bin/updated-faillint: go1.18.1
```

Could we release a version built with a newer version of Go?

**Comment by fatih** [comment]
@joshmgross sigh :/ I just updated the CI to use the newer versions. Let me do it quickly.

**Comment by fatih** [comment]
Just published a new one, this will build with 1.18: https://github.com/fatih/faillint/releases/tag/v1.10.0

**Comment by joshmgross** [comment]
Awesome thanks @fatih, confirmed that v1.10.0 is working
